config/default.yml

```diff
@@ -148,6 +148,71 @@ Style/BarePercentLiterals:
     - percent_q
     - bare_percent
 
+Style/Blocks:
+  EnforcedStyle: multiline
+  SupportedStyles:
+    # The `multiline` style enforces braces around single line blocks and
+    # do..end around multi-line blocks.
+    - multiline
```

Comment:
What would you suggest? If it's entirely dependent on the number of lines in the block, what's a pithy name for that style? Something about `lines`?